app/controllers/application_controller.rb

```diff
@@ -1,5 +1,8 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.first
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you add the `authenticate_user!` helper so that only logged in users can access our pages.

  ```
  before_action :authenticate_user!
  ```